plugins/mumble_plugin_linux.h

```diff
@@ -151,11 +148,45 @@ static inline PTR_TYPE_CONCRETE getModuleAddr(pid_t pid, const wchar_t *modname)
 	return 0;
 }
 
-static inline PTR_TYPE_CONCRETE getModuleAddr(const wchar_t *modname) {
+static inline procptr_t getModuleAddr(const wchar_t *modname) {
 	return getModuleAddr(pPid, modname);
 }
 
-static inline bool peekProc(PTR_TYPE base, void *dest, size_t len) {
+static inline int checkProcessIs64Bit(const pid_t pid)
+{
+	// We can know the process architecture by looking at its ELF header.
+	// The fifth byte is 1 in case the process is 32-bit or 2 in case it's 64-bit.
+	// The first 4 bytes constitute the magical number in ASCII: 0x7F 45 4c 46.
+
+	std::stringstream ss;
+	ss << std::string("/proc/");
```

Comment:
Why the std::string wrapping? Just causes a needless alloc. Pretty sure there's a const char* overload.

Reply:
I don't know, @mkrautz wrote the code.

I think that we should clean up both the Linux and Windows headers in a new pull request.

Reply:
I took the code from `getModuleAddr()`, which was written by @mkrautz.

How would you do it?

Reply:
```cpp
ss << "/proc/";
```
Should work I think.

Reply:
Just tried, it works.